### PR TITLE
Dilpreet | Test | Template change - setting for base currency hide/show, exchange rate, POS, POC, LUT 

### PR DESCRIPTION
**Pull request by dilpreetsinghofficial**

### CODE REVIEW

File: apps/web-giddh/src/app/gst/gst-setting/gst-setting.component.scss [hunk 1]
```diff
@@ -12,7 +12,7 @@ tr {
 
     &.invalid-row {
         td {
-            padding-bottom: 50px;
+            padding-bottom: 30px;
```
Comment (nisha-walkover) [inline apps/web-giddh/src/app/gst/gst-setting/gst-setting.component.scss:15]: add pb-3 class for padding remove css

File: apps/web-giddh/src/app/gst/gst-setting/gst-setting.component.ts [hunk 2]
```diff
@@ -54,21 +55,17 @@ export class GstSettingComponent implements OnInit, OnDestroy {
     /** Hold lut number item list*/
     public lutItemList: any[] = [];
 
+    public showHideLutForm: boolean;
```
Comment (nisha-walkover) [inline apps/web-giddh/src/app/gst/gst-setting/gst-setting.component.ts:58]: add documentation

File: apps/web-giddh/src/assets/locale/gst-setting/mr.json [hunk 3]
```diff
@@ -5,5 +5,7 @@
     "without_pay": "आईजीएसटी/एलयूटी बाँड भरल्याशिवाय",
     "lut_number":"LUT क्रमांक ",
     "enter_lut_number":"LUT क्रमांक प्रविष्ट करा",
-    "add_new_line": "+ नवीन ओळ जोडा"
+    "add_new_line": "+ नवीन ओळ जोडा",
+   "page_heading":"जीएसटी सेटिंग",
+    "confirm_delete_message": "Are you sure you want to delete lut number?"
```
Comment (nisha-walkover) [inline apps/web-giddh/src/assets/locale/gst-setting/mr.json]: check translation

File: apps/web-giddh/src/app/gst/gst-setting/gst-setting.component.scss (hunk 1 above)
Comment (dilpreetsinghofficial) [reply apps/web-giddh/src/app/gst/gst-setting/gst-setting.component.scss:15]: This class will add dynamically

**Review by nisha-walkover — APPROVED** [review]

File: apps/web-giddh/src/app/gst/gst-setting/gst-setting.component.ts [hunk 4]
```diff
@@ -237,29 +231,71 @@ export class GstSettingComponent implements OnInit, OnDestroy {
...
+    }
+
+    /**
+     * Open confirmation dialog for delete LUT number
+     *
+     * @private
+     * @param {*} request
+     * @memberof GstSettingComponent
+     */
+    private openConfirmationDialog(index: number): void {
```
Comment (dvCodeWorld) [inline apps/web-giddh/src/app/gst/gst-setting/gst-setting.component.ts:246]:
```suggestion
    private openConfirmationDialog(index: number): void || null {
```

File: apps/web-giddh/src/app/gst/gst-setting/gst-setting.component.ts [hunk 5]
```diff
@@ -237,29 +231,71 @@ export class GstSettingComponent implements OnInit, OnDestroy {
...
+                            }
+                        }
+                    } else {
+                        if (mappingForm.get('uniqueName')?.value) {
+                            if (this.responseArray?.length) {
+                                this.responseArray[index] = [];
+                            }
+                            this.componentStore.deleteLutNumber({ lutNumberUniqueName: mappingForm.get('uniqueName')?.value });
+                            mappings.removeAt(index);
+                        } else {
```
Comment (dvCodeWorld) [inline apps/web-giddh/src/app/gst/gst-setting/gst-setting.component.ts]: if (mappingForm.get('uniqueName')?.value) {
                            if (this.responseArray?.length) {
                                this.responseArray[index] = [];
                            }
                            this.componentStore.deleteLutNumber({ lutNumberUniqueName: mappingForm.get('uniqueName')?.value });  
                        } 
                            mappings.removeAt(index);

Comment (dvCodeWorld) [reply apps/web-giddh/src/app/gst/gst-setting/gst-setting.component.ts]: Keep this line "mappings.removeAt(index);"

out side the if condition and no need to write else block

**Review by dvCodeWorld — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/gst/gst-setting/gst-setting.component.ts (hunk 4 above)
Comment (dilpreetsinghofficial) [reply apps/web-giddh/src/app/gst/gst-setting/gst-setting.component.ts:246]: removing return

**Review by dvCodeWorld — APPROVED** [review]

**Review by ravinderwalkover — APPROVED** [review]

**Pull request merged**